### PR TITLE
feat: add tests to the Storybook package

### PR DESCRIPTION
**Pull request by jimhunty**
The storybook package didn't have any tests. Therefore I added tests along with Jest Configurator conventions and made sure they gave 100% coverage.

### CODE REVIEW

File: packages/storybook/package.json [hunk 1]
```diff
@@ -5,7 +5,11 @@
   "scripts": {
     "fmt": "prettier --write '**/*.*'",
     "prettier:diff": "prettier --list-different '**/*.*'",
-    "lint": "eslint . && yarn prettier:diff"
+    "lint": "eslint . && yarn prettier:diff",
+    "test:android": "jest --config='./__tests__/android/jest.config.js'",
+    "test:ios": "jest --config='./__tests__/ios/jest.config.js'",
+    "test:web": "jest --config='./__tests__/web/jest.config.js'",
+    "test": "yarn test:android -- --ci --bail --coverage && yarn test:ios -- --ci --bail --coverage && yarn test:web -- --ci --bail --coverage"
```
Comment (colincclark) [inline packages/storybook/package.json]: no need for the doube `--` - yarn does not need them anymore

Comment (jimhunty) [reply packages/storybook/package.json]: Should be sorted now.

Comment (colincclark) [reply packages/storybook/package.json]: did you commit?

Comment (jimhunty) [reply packages/storybook/package.json]: Forgot to push!

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/077b2d5b804280f755f99ca48098eb4d72dd02d3/dextrose-test/index.html

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/cb2ce93d16e85b5d4f1c3dec25621a4af9bd0c2b/dextrose-test/index.html

**Pull request merged**